### PR TITLE
Release v0.23.2

### PR DESCRIPTION
**Pull request by thanethomson**
:book: [CHANGELOG rendered](https://github.com/informalsystems/tendermint-rs/blob/release/v0.23.2/CHANGELOG.md#v0232)

* [ ] Referenced an issue explaining the need for the change
* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Added entry in `.changelog/`

## Blockers

* [x] #1047

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,49 @@
...
+  ([#1030](https://github.com/informalsystems/tendermint-rs/issues/1030))
+- `[tendermint]` Improve `tendermint::Time`
+  ([#1030](https://github.com/informalsystems/tendermint-rs/issues/1030)):
+  * Restrict the validity range of `Time` to dates with years in the range
+    1-9999, to match the specification of protobuf message `Timestamp`.
+    Add an `ErrorDetail` variant `DateOutOfRange` to report when this
+    restriction is not met.
+  * Added a conversion to, and a fallible conversion from,
+    `OffsetDateTime` of the `time` crate.
+  * Added `Time` methods `checked_add` and `checked_sub`.
```
Comment (mzabaluev) [inline CHANGELOG.md:45]: These have been removed by #1048.

Comment (thanethomson) [reply CHANGELOG.md]: Good catch! I forgot to rebuild the changelog.

**Comment by mzabaluev** [comment]
Checked with a pending PR to ibc-rs, looks good!

**Review by mzabaluev — APPROVED** [review]
LGTM

**Pull request merged**